### ISSUE TITLE
disable the clickfunction on backdrop close i need backdrop to stay back

### ISSUE THREAD

**Issue by jenkyjeni** [cipchk/ngx-bootstrap-modal#1]
disable the clickfunction on backdrop close i need backdrop to stay back

**Comment by cipchk** [comment]
a simple demo:

```typescript
this.dialogService.addDialog(AlertComponent, { message: 'Click outside only close button' }, { backdrop: 'static' });
```

could you give me more details?

**Comment by jenkyjeni** [comment]
Hi 
I need backdrop: 'static' in the confirm can you providce me an example --
Jenifer
 

    On Thursday, 24 August 2017 2:53 PM, 卡色 <notifications@github.com> wrote:
 

 a simple demo:this.dialogService.addDialog(AlertComponent, { message: 'Click outside only close button' }, { backdrop: 'static' });could you give me more details?—

**Comment by cipchk** [comment]
```typescript
this.dialogService.confirm('title', 'content', { backdrop: 'static' })
```

**Comment by jenkyjeni** [comment]
Hi 
Its throwing error builtInOptions interface doesn't have the variable backdrop.
below the error message
Argument of type '{backdrop: string}' is not assignable to parameter of type 'BuiltInOptions'. Object literal may only specify known properties, and 'backdrop' does not exist in type 'BuiltInOptions'.

 --
Jenifer
 

    On Saturday, 26 August 2017 11:47 AM, 卡色 <notifications@github.com> wrote:
 

 this.dialogService.confirm('title', 'content', { backdrop: 'static' })—

**Comment by cipchk** [comment]
bug confirm, thanks.

but, you can using method.

```typescript
this.dialogService.show({
    type: 'confirm',
    title: 'title',
    content: 'content',
    onHide: (res: boolean) => {
        console.log(res);
    }
}, {
    backdrop: 'static'
});
```

today fix this.

**Comment by cipchk** [comment]
fix 1.0.10

so, you can:

```typescript
this.dialogService.confirm('title', 'content', { backdrop: 'static' })
```

**Comment by jenkyjeni** [comment]
Hi 
Thanks for the Quick fix and Update --
Jenifer
 

    On Saturday, 26 August 2017 2:07 PM, 卡色 <notifications@github.com> wrote:
 

 Closed #1.—
